lib/ansible/modules/git.py

```diff
@@ -28,6 +28,8 @@
     dest:
         description:
             - The path of where the repository should be checked out. This
+              is similar to C(git clone repo_url target_name). The repository
+              named in C(repo) is not appended to this path. This
```

Comment:
Sorry, a few more minor clarifications after I read the related issue in detail.
```suggestion
              is equivalent to C(git clone [repo_url] [directory]). The repository
              named in C(repo) is not appended to this path and the destination directory must be empty. This
```